quickstarts/quickstart-template-guide.md

```diff
@@ -0,0 +1,141 @@
+
+A good quickstart answers the following questions:
+
+- Scope:
+ - What is the core purpose of this application?
+ - What is the minimal use case that is covered in this quickstart?
+- Install (if applicable):
+ - How do I download, install, and configure the application?
+ - How do I install and configure the application?
+ - How can I get any required access keys or authentication credentials?
+- Hello  World:
+ - How can I run a simple workflow for a core feature or a common use? case.
+- Next steps:
+ - What other features are available to explore in the application?
+
+A Quickstart guide is often confused with a Getting Started  guide. Though both these documents help the users get acquainted with the application they widely differ in their target audience. 
+
+There are instances where a Marketing guide is often confused with a Quickstart and a Getting Started guide. The inline table gives a brief comparison about the differences. 
+
+| |Quickstart Guide|Getting Started Guide| Marketing Guide|
+|--------|----------------|----------------------|---------------|
+ |Target Audience|Domain Experts: Know the problem space and are aware of what they need to do|Beginners:New to problem space and the product|Business Decision Maker: Makes strategic decisions on whether to buy the product 
+|Content|Minimal conceptual information|Detailed conceptual information on product/domain| No/less conceptual information. Instead focuses on the benefits and the customers using the product.
+|Focus|how-to| how-to and why|  sales
+
+## Why do I need a quickstart?
+
+A quickstart is often the first opportunity for your user to build an opinion about the technical quality of your project. It can:
+- Make users comfortable using the application.
+- Work with any application version the user might use.
+- Lead to the willingness to handle complex workflows.
+- Help the users in exploring additional features.
+- Improve the customer experience, and help reduce costs by lowering the number of support requests.
+
+## Tips for writing a quickstart
+
+- Highlight the primary feature of your application.
+- Document the quickest/easiest way to implement the primary feature  of your application.
+- Pick a use case that the user can complete within 1-2 hours with a preference for a shorter time.
+-  Avoid complicating the quickstart by including error scenarios/complex use cases.
+- Lengthy quickstarts can overwhelm users. Consider condensing or removing steps or reevaluating the scope of the quickstart.
+- For code samples, ensure that you include:  
+ - Any required `import` or `using` statements.
+ - Code comments that explain what the code does.
+
+## About the "Metadata" fields
+
+For more information on how to update the metadata, see [Document version](../base/base-guide.md#document-version), [Last Updated](../base/base-guide.md#last-updated), and [Application Version](../base/base-guide.md#application-versions) on the Good Docs Projects Base template.
+
```

Comment:
**Suggestion (blocking):** We have not released the base template and personally I don't think we should release it. (Cameron and I disagree about the base template.) Since it's not released, I recommend removing the metadata fields from your guide and from your template.

**Chore:** The template working group leaders (Cameron, Alyssa, Dee) need to meet to resolve the question about metadata because it's a point of disagreement between us. We need to get alignment and consensus about it.

Reply:
Fixed - We have removed the them from guide and template